Gemfile.lock

```diff
@@ -161,7 +161,7 @@ GEM
       capistrano (~> 3.7)
       capistrano-bundler
       puma (>= 4.0, < 6.0)
-    capybara (3.36.0)
+    capybara (3.37.1)
```

Comment:
### [capybara](https://github.com/teamcapybara/capybara)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.36.0 → 3.37.1

[_[change-log](https://github.com/teamcapybara/capybara/blob/master/History.md), [source-code](https://github.com/teamcapybara/capybara), [gem-diff](https://my.diffend.io/gems/capybara/3.36.0/3.37.1)_]





<details>
<summary>Commits</summary>

A change of **31** commits. See the full changes on [the compare page](https://github.com/teamcapybara/capybara/compare/3.36.0...3.37.1).

These are the first 10 commits:
- (0708e38) [Setting cert if no key prevents submission to rubygems](https://github.com/teamcapybara/capybara/commit/0708e38307aa696e8f53746c987dd4dadb593c23)
- (f46ae92) [Fixed typo](https://github.com/teamcapybara/capybara/commit/f46ae92f5a23661598f5289f51b2e9d60eab514b)
- (a12915a) [Merge pull request #2508 from hsbt/typofix](https://github.com/teamcapybara/capybara/commit/a12915a6ae6faf1f79445ccd9e2b29d4c3850d7a)
- (b9c9acc) [Info I needed this week](https://github.com/teamcapybara/capybara/commit/b9c9accc218429bdda3ba381f4e19ff79bc281dc)
- (57d4783) [Merge pull request #2509 from pablobm/doc-server-host](https://github.com/teamcapybara/capybara/commit/57d4783765c559c776df545c71a8519bca824c06)
- (e5d5b6f) [Fix typo Nokogiri::HTL5 -> Nokogiri::HTML5](https://github.com/teamcapybara/capybara/commit/e5d5b6f1fb80e6cfc8346d17f1a44d465cb610e3)
- (e704d00) [Merge pull request #2516 from hanachin/patch-1](https://github.com/teamcapybara/capybara/commit/e704d00879fb1d1e1a0cc01e04c101bcd8af4a68)
- (7add102) [Allow Selenium 4.1.0 or higher](https://github.com/teamcapybara/capybara/commit/7add1028e541abb7e88a08244021618e289a3fab)
- (a958240) [Merge pull request #2520 from JunichiIto/allow-selenium-4-1-…](https://github.com/teamcapybara/capybara/commit/a958240ef44b44ca90be8e7b8fbcba8511a6e262)
- (10115c8) [Document the text "rapid" mode in RubyDocs](https://github.com/teamcapybara/capybara/commit/10115c84f728f4f33d4744b6a7d36dc8b80a4161)


</details>